src/styles/reset.scss

```diff
@@ -0,0 +1,6 @@
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
```

Comment:
It is not recommended to set the font-family in a global reset file. Consider moving it to a more specific selector or create a base styling file for typography.

Reply:
html or body, for example